Framework/PythonInterface/plugins/algorithms/EnggFitTOFFromPeaks.py

```diff
@@ -92,25 +93,23 @@ def _fit_dSpacingTOF(self, fitted_peaks_table):
             raise ValueError('Cannot fit a quadratic function with less than three peaks. Got a table of '
                              + 'peaks with ' + str(num_peaks) + ' peaks')
 
-        convert_tbl_alg = self.createChildAlgorithm('ConvertTableToMatrixWorkspace')
-        convert_tbl_alg.setProperty('InputWorkspace', fitted_peaks_table)
-        convert_tbl_alg.setProperty('ColumnX', 'dSpacing')
-        convert_tbl_alg.setProperty('ColumnY', 'X0')
-        convert_tbl_alg.execute()
-        d_tof_conversion_ws = convert_tbl_alg.getProperty('OutputWorkspace').value
+        d_tof_conversion_ws = ConvertTableToMatrixWorkspace(
+            InputWorkspace=fitted_peaks_table,
+            ColumnX='dSpacing',
+            ColumnY='X0'
+        )
 
         # Fit the curve to get linear coefficients of TOF <-> dSpacing relationship for the detector
-        fit_alg = self.createChildAlgorithm('Fit')
-        fit_alg.setProperty('Function', 'name=Quadratic')
-        fit_alg.setProperty('InputWorkspace', d_tof_conversion_ws)
-        fit_alg.setProperty('WorkspaceIndex', 0)
-        fit_alg.setProperty('CreateOutput', True)
-        fit_alg.execute()
-        param_table = fit_alg.getProperty('OutputParameters').value
-
-        tzero = param_table.cell('Value', 0)  # A0
-        difc = param_table.cell('Value', 1)  # A1
-        difa = param_table.cell('Value', 2)  # A2
+        fit_output = Fit(
+            Function='name=Quadratic',
+            InputWorkspace=d_tof_conversion_ws,
+            WorkspaceIndex=0,
+            CreateOutput=True,
+        )
```

Comment:
And this call also leaves workspaces in the ADS, which is probably(?) not intended. Adding` StoreInADS=False` should restore previous behaviour.